input/pickle.go

```diff
@@ -34,7 +34,7 @@ func NewPickle(config cfg.Config, addr string, tbl *table.Table, bad *badmetrics
 func (p *Pickle) Handle(c net.Conn) {
 	defer c.Close()
 	// TODO c.SetTimeout(60e9)
-	r := bufio.NewReaderSize(c, 4096)
+	r := bufio.NewReader(c)
```

Comment:
before and after are equivalent as far as i can see. what's the intent here?

Reply:
You're right, I don't think it's necessary.  Originally I thought that it might be causing issues with packets larger than that.